docs/changelog-fragments/675.feature.rst

```diff
@@ -0,0 +1 @@
+Expose the SSH_OPTIONS_KEY_EXCHANGE option -- by :user:`NilashishC`
```

Comment:
Instead of having change notes in a TODO-list style, they should refer to things that have already changed since the previous release. Hence, it's usually in the past tense or similar. Also, it's RST so inline code can be highlighted:
```suggestion
The underlying ``SSH_OPTIONS_KEY_EXCHANGE`` option of ``libssh`` is
now available as ``key_exchange_algorithms`` -- by :user:`NilashishC`.
```
Also, single sentences with a byline at the end should have a trailing period, unlike detached standalone bylines.

You can then preview how it renders and see if it looks coherent: https://ansible-pylibssh--675.org.readthedocs.build/en/675/changelog/#features